src/index.js

```diff
@@ -1 +1,26 @@
 import './style.scss';
+import Request from "./modules/request";
+
+const coinsUrl='https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&order=market_cap_desc&per_page=100&page=1&sparkline=false';
+const url='https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/QzuepOdb4S7f14QxmbA0/scores';
+const request = new Request(coinsUrl);
+const req= new Request(url);
+const coins=document.querySelector('.coins');
```

Comment:
If this query was in the `addToDom`() , function would have less arguments passed into and code would be easier to read.